lib/scala/library-manager/src/test/scala/org/enso/librarymanager/test/published/repository/DummyRepository.scala

```diff
@@ -97,7 +97,7 @@ abstract class DummyRepository(toolsRootDirectory: Path) {
       namespace = lib.libraryName.namespace,
       version   = lib.version.toString()
     )
-    pkg.save()
+    pkg.save(true)
```

Comment:
What was the failure when `0.0.0-dev` wasn't stored into the `package.yaml`?

Reply:
I see.
```
sbt language-server/testOnly *LibrariesTest
```
fails on my computer. I saw the failure before, but never on the CI for my PRs, so I assumed it is some build mismatch problem.